From 000cd04ec9d2b496b62a6c582507f65797d6f1d1 Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Thu, 3 Jan 2013 09:00:17 -0800
Subject: [PATCH] watchdog: xilinxps: Sorting #includes

Sorting #includes for easier maintenance and removing a duplicate which already
sneaked in.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
Acked-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/watchdog/xilinx_wdtps.c |   15 +++++++--------
 1 files changed, 7 insertions(+), 8 deletions(-)

diff --git a/drivers/watchdog/xilinx_wdtps.c b/drivers/watchdog/xilinx_wdtps.c
index cf4c9a1..c43ca4b 100644
--- a/drivers/watchdog/xilinx_wdtps.c
+++ b/drivers/watchdog/xilinx_wdtps.c
@@ -15,21 +15,20 @@
  */
 
 #include <linux/export.h>
-#include <linux/module.h>
-#include <linux/io.h>
 #include <linux/fs.h>
 #include <linux/init.h>
 #include <linux/interrupt.h>
-#include <linux/kernel.h>
-#include <linux/reboot.h>
-#include <linux/uaccess.h>
-#include <linux/watchdog.h>
-#include <linux/interrupt.h>
+#include <linux/io.h>
 #include <linux/irq.h>
+#include <linux/kernel.h>
 #include <linux/miscdevice.h>
+#include <linux/module.h>
+#include <linux/of.h>
 #include <linux/platform_device.h>
+#include <linux/reboot.h>
 #include <linux/slab.h>
-#include <linux/of.h>
+#include <linux/uaccess.h>
+#include <linux/watchdog.h>
 
 #define XWDTPS_DEFAULT_TIMEOUT	10
 /* Supports 1 - 516 sec */
-- 
1.7.1

